src/componentes/mounted/busca-ativa/gestantes/APS/indicador_1/grafico_indicador_1_futuro.js

```
@@ -1,203 +1,294 @@
+				}}
+			>
+				{quadrisFuturosFormatados && `${quadrisFuturosFormatados} -`} Gestantes
+				com DUM preenchida por total de consultas e captação
+			</h2>
+			<ScoreCardGrid
+				valores={[
+					{
+						descricao:
+							"Gestantes com mais de 6 consultas (1ª consulta até a 12ª semana)",
+						valor: dataQuadriAtual.reduce((acumulador, item) => {
+							return (item.id_status_usuario == 8 ||
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)